topics/csharp/index.md

```diff
@@ -1,13 +1,16 @@
 ---
-aliases: c-sharp, csharp-code, csharp7, csharp-core, csharp6, csharp-resources
+aliases: c-sharp
```

Comment:
Are these aliases you are removing not typically used?

Reply:
Yes, I don't think so. However, I can't count the number of related repositories now. If there is any mistake I'll add them back later.

Reply:
I am generally 👍 on not overloading the aliases. So, I like this change.